### PR TITLE
Mudule set up

### PR DESCRIPTION
**Pull request by Booze33**
- Install linters
- run npm
- Install luxon
- create index files
- create modules with the js files
- import from modules to index.js
- Fix bugs

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import Library from './modules/library.js';
+import displaySection from './modules/displaySection.js';
+import { DateTime } from './node_modules/luxon/src/luxon.js';
+
+// let listBooks
+const listBooks = new Library();
+listBooks.className = 'purple';
+
+// ====================== NAVIGATION =========================
+function UpdateTime() {
```
Comment (ouasamine) [inline index.js]: - [ ] Please note that as part of this project's [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md) you need to refactor all your traditional functions to the new ES6 arrow syntax, which is not the case for this function, please convert it to an arrow function.

File: modules/displaySection.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+const sections = document.querySelectorAll('section');
+const pageTitle = document.querySelector('#page-title');
+// Menu Links
+const listLink = document.querySelector('#m-list a');
+const addLink = document.querySelector('#m-add a');
+const contactLink = document.querySelector('#m-contact a');
+
+function displaySection(sectionToDisp) {
+  sections.forEach((section) => {
```
Comment (ouasamine) [inline modules/displaySection.js]: - [ ] This function still needs to be refactored as well, please convert it to an arrow function.

**Review by ouasamine — CHANGES_REQUESTED** [review]
Hi @Booze33 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Nice use of ES6 import/export 👏🏽 
- Good work splitting your app logic from DOM manipulation 👍🏽 ✔️ 
- Nice PR title 👏🏽 , however, there is a typo in the PR title that needs to be fixed 😉 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @Booze33 ,

Great job so far! 

May I highlight the following: 
- [x] The project is set correctly✔️ 
- [x] No Linter errors✔️ 

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**